Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/WorkflowExecutionBusiness.java

```diff
@@ -54,11 +56,17 @@
     private static final Logger logger = Logger.getLogger(WorkflowExecutionBusiness.class);
     private WorkflowEngineInstantiator engine;
 
-    public WorkflowExecutionBusiness(String applicationClass) throws BusinessException {
+    public WorkflowExecutionBusiness(String engineEndpoint) throws BusinessException {
 
-        engine = WorkflowEngineInstantiator.create(applicationClass);
-    }
+        //HACK for testing while still having simulations launched with VIP 1.16.1; to be removed before getting in production or replaced with a proper constant
+        if(engineEndpoint == null){
+            logger.info("WorkflowExecutionBusiness, endpoint is null, setting it to http://data-manager.grid.creatis.insa-lyon.fr/cgi-bin/m2Server-gasw3.1/moteur_server");
+            engineEndpoint="http://data-manager.grid.creatis.insa-lyon.fr/cgi-bin/m2Server-gasw3.1/moteur_server";
+        }
```

Comment:
Oh, that's the fallback you were talking about. 
In this case it is, but we probably don't need it. I used it here only for testing purposes (simulations run with the production portal have no engine)